deploy_gem.sh

```diff
@@ -0,0 +1,8 @@
+gem install bump
```

Comment:
Please add shebang `#!/bin/bash`

Also, it would be good to use "strict" mode in bash, so the program will be failed if even one command has been failed.
It will also disallow to use undefined variables.
`set -euo pipefail`